interfaces/dom.idl

```diff
@@ -1,9 +1,14 @@
+// GENERATED CONTENT - DO NOT EDIT
```

Comment:
So I should no longer ask for updates to this file whenever we update DOM?

Reply:
Yep, but they're not yet regular (semi-manual) so feel free to ping me or @foolip if you'd like a more hasty update.